src/shared/components/alert-button/AlertButton.tsx

```diff
@@ -0,0 +1,52 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import AlertIcon from 'static/icons/icon_alert_circle.svg';
+
+import { useShowTooltip } from 'src/shared/hooks/useShowTooltip';
+
+import Tooltip from 'src/shared/components/tooltip/Tooltip';
+
+import styles from './AlertButton.scss';
+
+type Props = {
+  helpText: React.ReactNode;
```

Comment:
`helpText ` -> `description`?

Reply:
change it to `tooltipText`  which is a bit clearer

Reply:
The `text` part is confusing. By reading it, one would expect it to be a string; but it can be any React node, really. If Andrea changes her mind back, it may contain a button 🤷‍♂️ 

`tooltipContent`?

Reply:
Agreed with the new name. why should it be an optional prop? does that mean we can reuse the same component for showing just the icon?

Reply:
> why should it be an optional prop? does that mean we can reuse the same component for showing just the icon?

Yep.

Reply:
🤷‍♂️  we might. Would be nice if we did.

Reply:
ok will do it as part of this PR, will be  a good test of using the shared component.